.gitignore

```diff
@@ -189,3 +189,6 @@ apps/stable_diffusion/web/models/
 
 # Stencil annotators.
 stencil_annotator/
+
+# brevitas custom op lib
+apps/language_models/scripts/vicuna.py
```

Comment:
This doesn't seem correct. Can you confirm?

Reply:
if you add it here it wont be detected by git. You want to add it to the black format command line with a `--exclude` or something